.gitignore

```diff
@@ -13,3 +13,6 @@
 
 # Output of the go coverage tool, specifically when used with LiteIDE
 *.out
+
+# Downloaded binaries
+bin/
```

Comment:
Would you mind adding the new line here?

Reply:
done